phpcs.xml.dist

```diff
@@ -162,12 +158,9 @@
     </rule>
 
 
-    <!-- *********************************************************************************** -->
-    <!-- Require native type hints for all parameters, properties, and return types in tests -->
-    <!-- *********************************************************************************** -->
-    <rule ref="SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingNativeTypeHint">
-        <exclude-pattern>src</exclude-pattern>
-    </rule>
```

Comment:
In this case, we can remove `ParameterTypeHint` but `PropertyTypeHint` needs to stay because it's [PHP 7.4+](https://php.watch/versions/7.4/typed-properties), correct? Should we add a `<!-- Requires PHP 7.4 -->` comment below?

Reply:
The rules below not only require PHP 7.4 but would also cause BC breaks which is why they're disabled here. Using the native property type is controlled in the actual rule definition above - this section here silences the error even if that was enabled.